Jellyfin.Server/Migrations/Routines/MigrateLibraryDb.cs

```diff
@@ -97,242 +107,321 @@ public void Perform()
+                """;
+
+                foreach (SqliteDataReader dto in connection.Query(chapterQuery))
+                {
+                    var chapter = GetChapter(dto);
+                    dbContext.Chapters.Add(chapter);
+                }
 
-        foreach (var item in peopleCache)
-        {
-            dbContext.Peoples.Add(item.Value.Person);
-            dbContext.PeopleBaseItemMap.AddRange(item.Value.Items.DistinctBy(e => (e.ItemId, e.PeopleId)));
-        }
+                stopwatch.Stop();
+                connection.Close();
+                _logger.LogInformation("Processing {0} Chapters entries took {1}.", dbContext.Chapters.Local.Count, stopwatch.Elapsed);
+                commitTasks.Add(() => CommitChanges(dbContext, "Chapters"));
 
-        peopleCache.Clear();
+                return stopwatch.Elapsed;
+            }),
+            Task.Run(() =>
+            {
+                var dbContext = CreateDbContext();
+                using var connection = new SqliteConnection(dbPath);
+                connection.Open();
+                var stopwatch = new Stopwatch();
+                stopwatch.Start();
+                _logger.LogInformation("Start moving AncestorIds.");
+                const string ancestorIdsQuery = """
+                SELECT ItemId, AncestorId, AncestorIdText FROM AncestorIds
+                WHERE
+                EXISTS(SELECT 1 FROM TypedBaseItems WHERE TypedBaseItems.guid = AncestorIds.ItemId)
+                AND
+                EXISTS(SELECT 1 FROM TypedBaseItems WHERE TypedBaseItems.guid = AncestorIds.AncestorId)
+                """;
+
+                foreach (SqliteDataReader dto in connection.Query(ancestorIdsQuery))
+                {
+                    var ancestorId = GetAncestorId(dto);
+                    dbContext.AncestorIds.Add(ancestorId);
+                }
 
-        _logger.LogInformation("Try saving {0} People entries.", dbContext.MediaStreamInfos.Local.Count);
-        dbContext.SaveChanges();
-        migrationTotalTime += stopwatch.Elapsed;
-        _logger.LogInformation("Saving People entries took {0}.", stopwatch.Elapsed);
-        stopwatch.Restart();
+                stopwatch.Stop();
+                connection.Close();
+                _logger.LogInformation("Processing {0} AncestorIds entries took {1}.", dbContext.AncestorIds.Local.Count, stopwatch.Elapsed);
+                commitTasks.Add(() => CommitChanges(dbContext, "AncestorIds"));
 
-        _logger.LogInformation("Start moving Chapters.");
-        const string chapterQuery = """
-        SELECT ItemId,StartPositionTicks,Name,ImagePath,ImageDateModified,ChapterIndex from Chapters2
-        WHERE EXISTS(SELECT 1 FROM TypedBaseItems WHERE TypedBaseItems.guid = Chapters2.ItemId)
-        """;
-        dbContext.Chapters.ExecuteDelete();
+                return stopwatch.Elapsed;
+            })).GetAwaiter().GetResult();
```

Comment:
```suggestion
            })).ConfigureAwait(false).GetAwaiter().GetResult();
```

Shall I use ConfigureAwait here as well?

Reply:
that should not be nessesary here as no context switching is happening